crates/sequencer/src/deposit_data_mempool.rs

```diff
@@ -18,15 +14,6 @@ impl DepositDataMempool {
         }
     }
 
-    pub fn make_deposit_tx_from_data(&mut self, deposit_tx_data: Vec<u8>) -> TransactionRequest {
```

Comment:
again, we need this back. this is not doing abi decode, however, it's building a deposit transaction from the system signer with the given data from the request to be passed to eth_call